### DISCUSSION TITLE
ScriptReference/EditorUserBuildSettings-selectedBuildTargetGroup

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#226] [Notes]
# ScriptReference/EditorUserBuildSettings-selectedBuildTargetGroup

https://docs.unity3d.com/ScriptReference/EditorUserBuildSettings-selectedBuildTargetGroup.html

**Comment by mvi** [comment]
Note this is not the applied setting from switching platform, use [`activeBuildTarget`](https://docs.unity3d.com/ScriptReference/EditorUserBuildSettings-activeBuildTarget.html) instead for querying that. See the note on that page for how to get the active build target group too.  
`selectedBuildTargetGroup` instead is the setting currently being viewed in the EditorBuildSettings, and therefore may report a different target to the one you actually have active.
